src/plugins/plot/MctPlot.vue

```diff
@@ -484,7 +484,7 @@ export default {
                     end: range.max,
                     domain: this.config.xAxis.get('key')
                 })
-                    .then(this.stopLoading());
+                    .then(this.stopLoading.bind(this));
```

Comment:
Hmm. Don't we want to call `stopLoading` here?

Reply:
It was calling it right away which was stopping the loading too soon before the data was being loading. Binding it to this fixed the issue. I followed the example that was set by loadSeriesData above.